src/screens/Login/index.tsx

```diff
@@ -1,24 +1,71 @@
 import React from 'react'
 import { Image, View } from 'react-native'
 import RNBootSplash from 'react-native-bootsplash'
-import { ActivityIndicator, useTheme } from 'react-native-paper'
+import { ActivityIndicator, Paragraph, useTheme } from 'react-native-paper'
 import { GoogleSigninButton } from '@react-native-community/google-signin'
+import remoteConfig from '@react-native-firebase/remote-config'
+import semver from 'semver'
+import { CONFIG_CACHE_RATE, remoteConfiguration } from 'constants/remoteConfig'
 import { useStoreon } from 'storeon/react'
 import { Events, State } from 'store/types'
 import { AppTheme } from 'constants/theme'
-import { LOGIN } from 'store/auth'
+import { AUTH_EVENTS } from 'store/auth/events'
 import LogoSrc from 'assets/logo.png'
+import appConfig from '../../../app.json'
 
 import styles from './styles'
+import { LoginScreenProps } from './types'
 
-export const Login = () => {
+export const LoginScreen: React.FC<LoginScreenProps> = ({ navigation }) => {
   const { colors } = useTheme() as AppTheme
   const { dispatch, auth } = useStoreon<State, Events>('auth')
 
   // hide splash screen
   React.useEffect(() => {
-    RNBootSplash.hide({ fade: true })
-  }, [])
+    // Load remote configuration
+    const bootstrapAsync = async () => {
+      // Set defaults and fetch remote configuration
+      await remoteConfig().setDefaults(remoteConfiguration)
+      await remoteConfig().fetch(CONFIG_CACHE_RATE)
+      const fetchedRemotely = await remoteConfig().fetchAndActivate()
+
+      if (__DEV__) {
+        console.log('Configs retrieved from the backend and activated:', fetchedRemotely)
+      }
+    }
+
+    bootstrapAsync().finally(() => {
+      RNBootSplash.hide({ fade: true })
+
+      const latestVersion = remoteConfig().getValue('version').asString()
+      const newVersionAvailable = semver.gt(latestVersion, appConfig.version)
+
+      // Show a dialog to the user if we found that the user is not in a stable version
+      if (newVersionAvailable) {
+        navigation.navigate('Dialog', {
+          title: 'Update version',
+          confirmText: 'update',
+          onConfirm: async () => {
+            try {
+              // await Linking.openURL(Platform.OS === 'ios' ? storeLinks.ios : storeLinks.android)
```

Comment:
This is commented on purpose I left an example on how we can handle this when we have a first app in the stores.